src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -242,14 +263,25 @@ export default {
 				this.isAdmin = !!_get(data, 'my.isAdmin');
 				this.itemsInBasket = _map(_get(data, 'shop.basket.items.values'), 'id');
 				// Update the categories observer with the new setting, triggering updates
-				categoryObserver.setVariables({ ids: this.categoryIds });
+				categoryObserver.setVariables({
+					ids: this.realCategoryIds,
+					// @todo variables for fetching data for custom channels
+				});
 			},
 		});
 
 		// React to changes to the category data
 		categoryObserver.subscribe({
 			next: ({ data }) => {
-				this.categories = _get(data, 'lend.loanChannelsById') || [];
+				this.realCategories = _get(data, 'lend.loanChannelsById') || [];
+				this.customCategories = [];
+				this.customCategories.push({
+					id: 62,
+					name: 'Custom category',
+					loans: {
+						values: _get(data, 'lend.loans.values') || [],
+					},
+				});
```

Comment:
By pushing this "Structured" custom category data with the id, name and then loans object we should preserve the tracking setup.